src/test/java/edu/ucsb/cs156/happiercows/controllers/CommonsControllerTests.java

```diff
@@ -154,4 +156,54 @@ public void joinCommonsTest() throws Exception {
 
     assertEquals(responseString, cAsJson);
   }
+
+  @WithMockUser(roles = { "ADMIN" })
+  @Test
+  public void deleteCommons_test_admin_exists() throws Exception {
+      LocalDateTime someTime = LocalDateTime.parse("2022-03-05T15:50:10");
+      Commons c = Commons.builder()
+        .name("Jackson's Commons")
+        .cowPrice(500.99)
+        .milkPrice(8.99)
+        .startingBalance(1020.10)
+        .startingDate(someTime)
+        .build();
+      
+      when(commonsRepository.findById(eq(2L))).thenReturn(Optional.of(c));
+      doNothing().when(commonsRepository).deleteById(2L);
+      
+      MvcResult response = mockMvc.perform(
+              delete("/api/commons?id=2")
+                      .with(csrf()))
+              .andExpect(status().is(200)).andReturn();
+      
+      verify(commonsRepository, times(1)).findById(2L);
+      verify(commonsRepository, times(1)).deleteById(2L);
+      String responseString = response.getResponse().getContentAsString();
+      
+      assertEquals("commons with id 2 deleted", responseString);
+  }
+
+  @WithMockUser(roles = { "ADMIN" })
+  @Test
+  public void deleteCommons_test_admin_nonexists() throws Exception {
+      
+      when(commonsRepository.findById(eq(2L))).thenReturn(Optional.empty());
+      
+      MvcResult response = mockMvc.perform(
+              delete("/api/commons?id=2")
+                      .with(csrf()))
+              .andExpect(status().is(404)).andReturn();
+      
+      verify(commonsRepository, times(1)).findById(2L);
+      
+
+      String responseString = response.getResponse().getContentAsString();
+      
+      String expectedString = "{\"message\":\"Commons with id 2 not found\",\"type\":\"EntityNotFoundException\"}";
```

Comment:
If we're okay not implementing [HTTP DELETE to spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE), this looks good. For now, let's merge and focus on getting the commons list built.

Reply:
i think its fine foe now